### ISSUE TITLE
Quantumsuit Rebalance

### ISSUE THREAD

**Issue by Shattered2909** [GTNewHorizons/NewHorizons#4975]
The Quantum Suit is a late IV/early LUV tier armor that currently requires an Assline to craft even tho it only takes IV components and is made from Tungstensteel/Iridium.
While its power is comparable to a lot of other sets (Thaum, Blood Magic, even Advanced Nano) it is disproportionately expensive and now that the Assembling Machine has 9 slots, we can move the Quantumsuit recipe from the Assline to IV+ Assembling. The Assline recipe for the Quantumsuit takes molten Titanium, Soldering Alloy and Glowstone, we could remove 2 or even all 3 of those as the Suit itself takes IV tier materials and EV or lower doesn't make that much of a difference.

TLDR: Quantumsuit takes IV parts, why require an Assline? Just move to IV Assembler. It's not even that good.

**Comment by observeroftime02** [comment]
Sure, it's not a bad idea anyways. The quantum suit is not all that great, and if it fits the IV tier then it should be made in an assembler.
